README.md

```diff
@@ -70,7 +74,199 @@ bundle install
+
+### 1. Get a BigQuery project setup and add initial owners
+
+Ask in Slack on the `#twd_data_insights` channel for someone to help you
+procure a BigQuery instance in the `digital.education.gov.uk` Google Cloud
+Organisation.
+
+Ask for your `@digital.education.gov.uk` Google account to be setup as an owner
+via the IAM and Admin settings. Add other team members as necessary.
+
+#### Set up billing
+
+You also need to set up your BigQuery instance with paid billing. This is
+because `dfe-analytics` uses streaming, and streaming isn't allowed in the free
+tier:
+
+```bash
+accessDenied: Access Denied: BigQuery BigQuery: Streaming insert is not allowed
+in the free tier
+```
+
+### 2. Create a dataset and table
+
+You should create separate datasets for each environment (dev, qa, preprod, prod etc.).
+
+1. Open your project's BigQuery instance
+2. Go to the Analysis -> SQL Workspace section
+3. Tap on the 3 dots next to the project name, "Create data set"
+4. Name it something like `APPLICATIONNAME_events_ENVIRONMENT`, such as `applyforqts_events_production`, and set the location to `europe-west2 (London)`
+5. Select your new dataset
+6. Open a new query execution tab.
+7. Paste and run [create-events-table.sql](https://github.com/DFE-Digital/dfe-analytics/create-events-table.sql) to create a blank events table for dfe-analytics to stream data into (editing as appropriate).
+
+### 4. Create custom roles
+
+1. Go to IAM and Admin settings > Roles
+1. Click on "+ Create role"
+1. Create the 3 roles outlined below
+
+#### Analyst
+
+| Field             | Value                                              |
+| ----------------- | -------------------------------------------------- |
+| Title             | **BigQuery Analyst Custom**                        |
+| Description       | Assigned to accounts used by analysts and SQL developers. |
+| ID                | `bigquery_analyst_custom`                          |
+| Role launch stage | General Availability                               |
+| + Add permissions | See below                                          |
+
+<details>
+<summary>Permissions for bigquery_analyst_custom</summary>
+    bigquery.datasets.get
+    bigquery.datasets.getIamPolicy
+    bigquery.datasets.updateTag
+    bigquery.jobs.create
+    bigquery.jobs.get
+    bigquery.jobs.list
+    bigquery.jobs.listAll
+    bigquery.models.export
+    bigquery.models.getData
+    bigquery.models.getMetadata
+    bigquery.models.list
+    bigquery.routines.get
+    bigquery.routines.list
+    bigquery.savedqueries.create
+    bigquery.savedqueries.delete
+    bigquery.savedqueries.get
+    bigquery.savedqueries.list
+    bigquery.savedqueries.update
+    bigquery.tables.createSnapshot
+    bigquery.tables.export
+    bigquery.tables.get
+    bigquery.tables.getData
+    bigquery.tables.getIamPolicy
+    bigquery.tables.list
+    bigquery.tables.restoreSnapshot
+    resourcemanager.projects.get
```

Comment:
These seems to render out as one long list inside this `<details>` block, I think you need to use HTML here instead of markup, so maybe wrap this list of permissions up in a `<pre>...</pre>` block?

Reply:
```suggestion
<pre>
bigquery.datasets.get
bigquery.datasets.getIamPolicy
bigquery.datasets.updateTag
bigquery.jobs.create
bigquery.jobs.get
bigquery.jobs.list
bigquery.jobs.listAll
bigquery.models.export
bigquery.models.getData
bigquery.models.getMetadata
bigquery.models.list
bigquery.routines.get
bigquery.routines.list
bigquery.savedqueries.create
bigquery.savedqueries.delete
bigquery.savedqueries.get
bigquery.savedqueries.list
bigquery.savedqueries.update
bigquery.tables.createSnapshot
bigquery.tables.export
bigquery.tables.get
bigquery.tables.getData
bigquery.tables.getIamPolicy
bigquery.tables.list
bigquery.tables.restoreSnapshot
resourcemanager.projects.get
</pre>
```